src/components/Modal/Modal.styles.js

```diff
@@ -167,6 +167,28 @@ export const StyledModal = styled(ReactModalAdapter).attrs({
     width: 100vw;
     max-height: 100vh;
 
+    @media screen and (prefers-reduced-motion: reduce) {
```

Comment:
Pareil ici. Juste à appliquer `@media screen and (prefers-reduced-motion: reduce)` ou `@media screen and (prefers-reduced-motion: no-preference)` sur la propriété `transition` uniquement 
```suggestion
    ...
    @media screen and (prefers-reduced-motion: reduce) {
      transition: none;
    }
    
    // OU BIEN
    @media screen and (prefers-reduced-motion: no-preference) {
      transition: opacity ${speed.default};
    }
```

Reply:
J'ai juste changer les couleurs, le code avec les prefers-reduced-motion m'est complètement étranger alors je ne suis pas à l'aise de le modifier...